go/api/TypeDBDriver.go

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package api
+
+import (
+	"typedb_driver/go/api/database"
+	"typedb_driver/go/api/user"
+	typedb_driver "typedb_driver/go_wrapper"
```

Comment:
What does `typedb_driver` before the package do here? Is it needed?

Reply:
Looks like an alias for `typedb_driver.Session`? But I feel like it's a native session, and we don't want to return it from `Session()` call. Let's just remove it if it's not correct

Reply:
Or you can comment it out like `// Session()` below, it's fine